packages/expo-permissions/android/src/main/java/expo/modules/permissions/PermissionsModule.kt

```diff
@@ -18,6 +18,8 @@ import org.unimodules.interfaces.permissions.Permissions
 import org.unimodules.interfaces.permissions.PermissionsResponse
 import org.unimodules.interfaces.permissions.PermissionsResponseListener
 
+internal const val ERROR_TAG = "E_PERMISSIONS"
```

Comment:
Start with `ERR`, but it would be better to have more concrete tag than generic one that may be confusing.
```suggestion
internal const val ERROR_TAG = "ERR_PERMISSIONS"
```